proxy/envoy/config.go

```diff
@@ -269,6 +270,22 @@ func buildOutboundHTTPRoutes(instances []*model.ServiceInstance, services []*mod
 					routes = append(routes, buildDefaultRoute(cluster))
 				}
 
+				if service.External {
+					for _, route := range routes {
+						route.HostRewrite = service.Hostname
```

Comment:
OK, I see. I don't think "httpbin.org" should be in that list. This doesn't sound right @rshriram